sdk/communication/AzureCommunicationChat/Package.swift

```diff
@@ -48,7 +48,7 @@ let package = Package(
         .package(
             name: "Trouter",
             url: "https://github.com/microsoft/trouter-client-ios.git",
-            .branch("v0.0.1-beta.5")
+            from: "0.1.0"
```

Comment:
This should use `.exact("0.1.0")`. Otherwise a breaking change to trouter could be released that chat is not compatible with.

Reply:
Updated, thanks